iOS/src/toga_iOS/container.py

```diff
@@ -65,6 +65,7 @@ def __init__(self, content=None, layout_native=None, on_refresh=None):
         super().__init__(content=content, on_refresh=on_refresh)
         self.native = UIView.alloc().init()
         self.native.translatesAutoresizingMaskIntoConstraints = True
+        self.native.autoresizingMask = 63
```

Comment:
What's the significance of this magic number? I presume it's the manifestation of a bunch of internal constants? If so, preference would be to preserve those constants in `toga_iOS.libs` and reference them here.

Reply:
Yeah, makes sense.  Made the changes you suggested.